tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -121,24 +148,44 @@ func newPromProxy(namespace string) *promProxy {
 	}
 }
 
-func (p *promProxy) Setup() error {
+// portForward sets up local port forward to the pod specified by the "app" label
+func (p *promProxy) portForward(labelSelector string, localPort string, remotePort string) error {
 	var pod string
 	var err error
-	glog.Info("Setting up prometheus proxy")
-	getName := fmt.Sprintf("kubectl -n %s get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}'", p.namespace)
+
+	glog.Infof("Setting up %s proxy", labelSelector)
+	getName := fmt.Sprintf("kubectl -n %s get pod -l %s -o jsonpath='{.items[0].metadata.name}'", p.namespace, labelSelector)
 	pod, err = util.Shell(getName)
 	if err != nil {
 		return err
 	}
-	glog.Info("prometheus pod name: ", pod)
-	portFwdCmd := fmt.Sprintf("kubectl port-forward %s %s:%s -n %s", strings.Trim(pod, "'"), prometheusPort, prometheusPort, p.namespace)
+	glog.Infof("%s pod name: %s", labelSelector, pod)
+	portFwdCmd := fmt.Sprintf("kubectl port-forward %s %s:%s -n %s", strings.Trim(pod, "'"), localPort, remotePort, p.namespace)
 	glog.Info(portFwdCmd)
 	if p.portFwdProcess, err = util.RunBackground(portFwdCmd); err != nil {
 		glog.Errorf("Failed to port forward: %s", err)
 		return err
 	}
-	glog.Infof("running prometheus port-forward in background, pid = %d", p.portFwdProcess.Pid)
-	return err
+	glog.Infof("running %s port-forward in background, pid = %d", labelSelector, p.portFwdProcess.Pid)
+	return nil
+}
+
+func (p *promProxy) Setup() error {
+	var err error
+
+	if err = p.portForward("app=prometheus", prometheusPort, prometheusPort); err != nil {
+		return err
+	}
+
+	if err = p.portForward("istio=mixer", mixerMetricsPort, mixerMetricsPort); err != nil {
+		return err
+	}
+
+	if err = p.portForward("app=productpage", productPagePort, "80"); err != nil {
```

Comment:
add a comment that this forward is just to test the app liveness directly, initially was worried we weren't doing e2e anymore